src/main/java/org/openshift/jenkins/plugins/openshiftlogin/OpenShiftOAuth2SecurityRealm.java

```diff
@@ -257,11 +258,13 @@ public void initialize(HttpRequest request) throws IOException {
                                 }
                             });
                     GenericUrl url = new GenericUrl(serverPrefix + "/oapi/v1/users/~");
-
+                    
+                    // do self-sar here to see if you have the "admin" or "view" verb in the "build.openshift.io" api group on the "jenkins" resource 
```

Comment:
@deads2k since I didn't find references to the string "build.openshift.io" in the origin repo, I'm assuming that api group is not available yet.  Is that correct?

Would the url start with `/oapi/build.openshift.io/v1` ? .... analogous to `/apis/extension/v1` perhaps ...